nextcord/guild.py

```diff
@@ -515,6 +516,8 @@ def _from_data(self, guild: GuildPayload) -> None:
         for event in guild.get("guild_scheduled_events") or []:
             self._store_scheduled_event(event)
 
+        self._premium_progress_bar_enabled: bool = guild.get("premium_progress_bar_enabled", False)
```

Comment:
```suggestion
        self._premium_progress_bar_enabled: bool = guild["premium_progress_bar_enabled"]
```
Seeing as this return `true` or `false` and not `null`, the `.get` isn't really needed